opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -284,14 +322,19 @@ class Translator {
     /**
      * Internal helper method which makes the actual request to the wink service
      */
-    _makeRequest(path, method, content) {
+    _makeRequest(path, method, content, includeBearerHeader) {
+       
+        var addBearerHeader = (typeof includeBearerHeader !== 'undefined') ?  includeBearerHeader : true;
```

Comment:
i see your point -- update coming up

---
In reply to: [89002893](https://github.com/openT2T/translators/pull/126#discussion_r89002893) [](ancestors = 89002893,88971888)